stdeb.cfg

```diff
@@ -6,7 +6,7 @@ Depends:
     ipython (>= 2.0),
     python-traits,
     python-traitsui,
-    python-nose,
+    python-pytest,
     ipython-qtconsole,
     ipython-notebook,
     python-h5py,
```

Comment:
all of them should be python3? It is going to be so quite to the future on debian/ ubuntu and derivatives...